src/app/components/modal/modalOpen.tsx

```
@@ -102,6 +108,21 @@ const ModalOpen = (props: {
                 </>
               )}
+              {props.mTarget === 'Notice' && (
+                <>
+                  {props.noticeTitle && (
+                    <h3 className="font-bold text-lg mb-3">
+                      {props.noticeTitle}
+                    </h3>
+                  )}
+                  <div
+                    className="whitespace-pre-wrap"
+                    dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)